spec/unit/generator/embed_frameworks_script_spec.rb

```diff
@@ -3,15 +3,14 @@
 module Pod
   describe Generator::EmbedFrameworksScript do
     it 'returns the embed frameworks script' do
-      target_definition = Podfile::TargetDefinition.new(:default, nil)
       frameworks = {
-        'Debug'   => %w(Loopback.framework Reveal.framework),
+        'Debug'   => %w(Pods/Loopback.framework Reveal.framework),
```

Comment:
I don't see any advantage of a separate test for that. The history documents why this change was introduced. It's still a legit test scenario.